study-builder/tenants/circadia/pages/password_reset.html

```diff
@@ -0,0 +1,191 @@
+      rel="stylesheet"
+    />
+
+    <style type="text/css">
+      body,
+      html {
+        padding: 0;
+        margin: 0;
+      }
+      .table {
+        display: table;
+        position: absolute;
+        height: 100%;
+        width: 100%;
+        background: linear-gradient(
+          rgba(255, 255, 255, 0.3),
+          rgba(255, 255, 255, 0)
+        );
+        background-color: #e8ebef;
+      }
+      .cell {
+        display: table-cell;
+        vertical-align: middle;
+      }
+      .content {
+        padding: 25px 0;
+        margin-left: auto;
+        margin-right: auto;
+        width: 280px;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="table">
+      <div class="cell">
+        <div class="content">
+          <!-- WIDGET -->
+          <div id="change-password-widget-container"></div>
+          <!-- END WIDGET -->
+        </div>
+      </div>
+    </div>
+
+    <script src="https://cdn.auth0.com/js/change-password-1.1.min.js"></script>
+
+    <style type="text/css">
+      body {
+        font-family: 'Nunito', sans-serif;
+      }
+
+      .auth0-lock.auth0-lock .auth0-lock-overlay {
+        background: radial-gradient(#ffffff, #f3f3f2, #f3f3f2);
+        background-image: radial-gradient(#ffffff, #f3f3f2, #f3f3f2);
+        background-color: #fff;
+      }
+
+      .auth0-lock.auth0-lock.auth0-lock-opened .auth0-lock-widget {
+        box-shadow: 0 0 40px 4px #888888;
+      }
+      .auth0-lock-name {
+        font-size: 12pt !important;
+        text-align: left;
+        padding-left: 0 !important;
+      }
+
+      .auth0-lock-content {
+        padding-top: 10px !important;
+      }
+
+      .auth0-lock-header .auth0-lock-header-bg {
+        background-color: white !important;
+      }
+
+      .auth0-lock.auth0-lock .auth0-lock-form p {
+        text-align: left;
+      }
+
+      .auth0-lock-name {
+        text-align: center;
+      }
+
+      .auth0-lock-header {
+        height: 110px !important;
+      }
+
+      .auth0-lock-header-bg {
+        height: 140px !important;
+      }
+
+      a:link {
+        text-decoration: underline !important;
+      }
+
+      #ddp-header {
+        padding-top: 10px;
+      }
+
+      #ddp-header .auth0-lock-header-logo {
+        max-width: 100%;
+        height: 60px;
+        margin-bottom: 10px;
+      }
+    </style>
+
+    <script>
+      var getQueryParamValue = function (paramName) {
+        var tokens = location.search.split(/&|#|\?/);
+        var queryParamPrefix = paramName + '=';
+        for (var i = 0; i < tokens.length; i++) {
+          if (tokens[i].indexOf(queryParamPrefix) == 0) {
+            return tokens[i].substring(queryParamPrefix.length);
+          }
+        }
+        return null;
+      };
+
+      var study_guid = getQueryParamValue('study');
+      var studyColor = '#0b1245'; // button primary color
+      var headerText = '';
+      var headerLogo = '##ASSETS_URL##/auth0/logo.png';
```

Comment:
Where is `ASSETS_URL` coming from? I don't see it in `circadia-config.json.ctmpl`?